openssl/src/x509/store.rs

```diff
@@ -211,6 +211,24 @@ impl X509LookupRef<File> {
             .map(|_| ())
         }
     }
+
+    #[corresponds(X509_load_crl_file)]
+    /// Specifies a file from which certificate revocation lists will be loaded
+    pub fn load_crl_file<P: AsRef<Path>>(
+        &mut self,
+        file: P,
+        file_type: SslFiletype,
+    ) -> Result<(), ErrorStack> {
```

Comment:
Probably makes sense to propagate the return value since it indicates the number of loaded CRL entries.

Reply:
I second that. However, the corresponding cert function does also not return that value, so I kept it returning `void`

I can change `load_cert_file` to returning ` Result<i32, ErrorStack` as well If that is OK for you.

Reply:
That'd be a breaking change so I wouldn't want to do it now. Can you just add a FIXME to change that signature to match load_crl_file?

Reply:
OK I'll do that